plugins/att_cybersecurity_alienvault_otx/komand_att_cybersecurity_alienvault_otx/actions/get_indicator_details/action.py

```diff
@@ -27,21 +27,21 @@ def run(self, params={}):
             "http_scans": {},
         }
 
-        indicator_type = get_indicatortypes(params.get(Input.INDICATOR_TYPE))
+        indicator_type = get_indicator_type(params.get(Input.INDICATOR_TYPE))
         indicator = params.get(Input.INDICATOR)
         section = params.get(Input.SECTION)
-        if section == "full":
-            results = self.connection.client.get_indicator_details_full(
-                indicator_type=indicator_type, indicator=indicator
-            )
-            clean_results = komand.helper.clean(results)
-            return {Output.RESULTS: clean_results}
 
-        results = self.connection.client.get_indicator_details_by_section(
+        if section == "full":
+            return {
+                Output.RESULTS: insightconnect_plugin_runtime.helper.clean(
+                    self.connection.client.get_indicator_details_full(
+                        indicator_type=indicator_type, indicator=indicator
+                    )
+                )
+            }
+
+        results[section] = self.connection.client.get_indicator_details_by_section(
             indicator_type=indicator_type, indicator=indicator, section=section
         )
-        output_template[section] = results
-
-        clean_results = komand.helper.clean(output_template)
 
-        return {Output.RESULTS: clean_results}
+        return {Output.RESULTS: insightconnect_plugin_runtime.helper.clean(results)}
```

Comment:
Why do we run the helper.clean on each results object, and then again on the final result object?

Reply:
We don't do this, both uses of helper.clean apply to different results. In the first case, when `section` is given as `full` and the `get_indicator_details_full` method is used. In the second case, when `get_indicator_details_by_section` method will be used for the remaining `section` values.